vast/extension.go

```diff
@@ -3,5 +3,16 @@ package vast
 // Extension type represents an <Extension> element that contains arbitrary XML data provided by the
 // platform to extend the VAST response.
 type Extension struct {
-	Data []byte `xml:",innerxml"`
+	Data          []byte          `xml:",innerxml"`
+	Verifications []*Verification `xml:"AdVerifications>Verification,omitempty"`
+}
+
+type Verification struct {
+	Vendor             string              `xml:"vendor,attr"`
+	ViewableImpression *ViewableImpression `xml:"ViewableImpression"`
```

Comment:
I think this needs to be an array of impressions...  there can be multiple `ViewableImpression` tags in the VAST

Reply:
oh my bad, I read it incorrectly. this is correct